dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst/csharp/Entitlements.plist

```diff
@@ -2,5 +2,14 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
     <dict>
+        <!-- app.sandbox is required by the App Store.-->
```

Comment:
I wonder if we can link to https://developer.apple.com/documentation/security/app_sandbox somewhere